log/sequencer.go

```diff
@@ -401,54 +400,53 @@ func (s Sequencer) SequenceBatch(ctx context.Context, logID int64, limit int, gu
 	}
 
 	seqCounter.Add(float64(numLeaves), label)
-	glog.Infof("%v: sequenced %v leaves, size %v, tree-revision %v", logID, numLeaves, newLogRoot.TreeSize, newLogRoot.TreeRevision)
+	if newLogRoot != nil {
+		glog.Infof("%v: sequenced %v leaves, size %v, tree-revision %v", logID, numLeaves, newLogRoot.TreeSize, newLogRoot.TreeRevision)
+	} else {
+		glog.Info("newLogRoot = nil")
```

Comment:
I think this shouldn't be reachable, so maybe make it `glog.Error`?

Reply:
Done